protocol/contracts/dao/Comptroller.sol

```diff
@@ -71,19 +71,30 @@ contract Comptroller is Setters {
     }
 
     function increaseSupply(uint256 newSupply) internal returns (uint256, uint256, uint256) {
-        (uint256 newRedeemable, uint256 lessDebt) = (0, 0);
+        (uint256 newRedeemable, uint256 lessDebt, uint256 poolReward) = (0, 0, 0);
 
         // 1. True up redeemable pool
         uint256 totalRedeemable = totalRedeemable();
         uint256 totalCoupons = totalCoupons();
         if (totalRedeemable < totalCoupons) {
+
+            // Get new redeemable coupons
             newRedeemable = totalCoupons.sub(totalRedeemable);
+            // Pad with Pool's potential cut
+            newRedeemable = newRedeemable.mul(100).div(SafeMath.sub(100, Constants.getOraclePoolRatio()));
```

Comment:
I don't think it is right.. Take `newRedeemable` as 100, then follow the PR's logic, 
```
newRedeemable = newRedeemable.mul(100).div(SafeMath.sub(100, Constants.getOraclePoolRatio())) = 100 * 100 / 80 = 125;
pollReward = newRedeemable.mul(Constants.getOraclePoolRatio()).div(100) = 125 * 0.2 = 25;
newRedeemable = newRedeemable.sub(poolReward) = 100 - 25 = 75;
```
So we will distribute 75% to coupon holder, it is different from what we want(80%). 
The correct logic should be:
```
 newRedeemable = totalCoupons.sub(totalRedeemable);
 newRedeemable = newRedeemable > newSupply ? newSupply : newRedeemable;
 poolReward = newRedeemable.mul(Constants.getOraclePoolRatio()).div(100);
 // Determine Redeemable's final cut
 newRedeemable = newRedeemable.sub(poolReward);
```

Reply:
Hi @JackZheng12, so from the code, if we had 100 newRedeemable, the outcomes there would be 25 ESD to the reward pool, and 100 to newRedeemable. The percentages would be:

Pool = 25/(100+25) * 100 = 20%
newRedeemable = 100/(100+25) * 100 = 80%

This is assuming that there was at least 125ESD newSupply.

Reply:
Oh, yes, you are right.  The algorithm is correct, if `newSupply` < `1.25 * origin_newRedeemable `, then we distribute 20% of newSupply to `LP`.  Thanks for your reply.

Reply:
You're welcome!

Reply:
You're welcome!